examples/hellogrpc/e2e-test.sh

```diff
@@ -116,6 +116,11 @@ edit() {
     logfail "./examples/hellogrpc/$1/server/edit.sh" "$2"
 }
 
+diff() {
+    logfail bazel build "examples/hellogrpc/$1/server:staging.diff"
+    logfail "bazel-bin/examples/hellogrpc/$1/server/staging.diff"
```

Comment:
is there a reason these don't use `bazel run`, which the hellohttp ones do? it doesn't seem like it's running in parallel so there shouldn't be lock contention. Is part of it just to test that direct execution works from outside the runfiles directory?

Reply:
Right the goal is to ensure that both `bazel run //whatever` and `bazel build //whatever && bazel-bin/whatever` work correctly.

Since these are e2es it would be too slow to do both, so we just pick one for hellohttp and the other for hellogrpc

Reply:
Gotcha. I'll add a comment in there in case someone else wonders why